src/device/router_device_worker.erl

```diff
@@ -962,7 +963,7 @@ handle_info(
     Device1 = router_device:metadata(Metadata, Device),
     DownlinkPacket = blockchain_helium_packet_v1:new_downlink(
         craft_join_reply(Device1, JoinAcceptArgs),
-        lorawan_mac_region:downlink_signal_strength(Region, TxFreq),
+        lora_plan:downlink_eirp(lora_plan:region_to_plan(Region), TxFreq),
```

Comment:
This is one where I'm wondering if we should keep the `downlink_signal_strength` name. Or alias it at the bottom of this file somewhere. 

Going to the spec for making a new downlink gets you
```erlang
-spec new_downlink(Payload :: binary(), SignalStrength :: integer(), Timestamp :: non_neg_integer(),
                   Frequency :: float(), DataRate :: string(), Rx2 :: window() | undefined) -> packet().
```

You kind of have to just _know_ what EIRP is already if you come across this.

Reply:
I've added a new API named max_tx_power.  We'll replace downlink_eirp with max_tx_power.

Reply:
Replaced with max_tx_power.